preferences.yaml

```diff
@@ -0,0 +1,76 @@
+swagger: '2.0'
+info:
+  version: 0.0.1
+  title: Preferences API
+  description: >-
+    The API lets you manage attributes associated with users (user preferences).
+
+    You can get any attribute for any user (read only) according to the rights
+    you owe on the user resource. 
+
+    There rights are set up at moment you've subscribed the service exposed by
+    this API.
+  license:
+    name: MIT
+host: localhost
+basePath: /api/v1
+schemes:
+  - https
+paths:
+  '/users/{fiscal_number}/preferences':
+    x-swagger-router-controller: Preferences
+    get:
+      operationId: getUserPreferences
+      summary: Get User's Preferences
+      description: Retrieve one user's preferences (ie. email and/or preferred language).
+      responses:
+        '200':
+          description: Found.
+          schema:
+            $ref: '#/definitions/User'
+          examples: {}
+          headers:
+            ETag:
```

Comment:
this (in tandem with `If-Match`) is useful only to avoid update conflicts - since this spec doesn't cover updates, what's the use of ETags here?